### PR TITLE
WIP: feature/katacodaDockerCompose

### PR DESCRIPTION
**Pull request by MarcelDiessner**
#67

### CODE REVIEW

**Comment by maybeec** [comment]
@TheMarcy, would be nice if you provide the issues this pr addresses in the pr comment simply by naming the issue number prefixed by a hash to make github properly link the pr to the ticket

File: runners/katacoda/index.ts
Comment (maybeec) [inline runners/katacoda/index.ts:212]: miss the assertion accordingly checking that docker compose completes.

@GuentherJulian is there also a ways similar to runJavaServer to wait until the success has been reported? Otherwise, you could not step ahead and access the potential URL started by docker compose.

**Review by maybeec — DISMISSED** [review]

File: runners/katacoda/index.ts
Comment (GuentherJulian) [reply runners/katacoda/index.ts:212]: @maybeec I think you refer to the console runner? I think we can do this the way as implemented in runServerJava for the console runner. So better wait until this pr is merged to test it.

File: runners/katacoda/index.ts
Comment (GuentherJulian) [inline runners/katacoda/index.ts:350]: You do not need the useDevonCommand parameter as docker is not included in the ide

File: runners/katacoda/templates/dockerCompose.md
Comment (GuentherJulian) [inline runners/katacoda/templates/dockerCompose.md:3]: This line can be removed. Katacoda displays the text from the index.json as the title. So it would be displayed twice

File: runners/katacoda/templates/dockerCompose.md
Comment (GuentherJulian) [inline runners/katacoda/templates/dockerCompose.md:10]: Here we can additionally mention that docker is stopped and restarted automatically by clicking on the command

File: runners/katacoda/templates/dockerCompose.md
Comment (GuentherJulian) [inline runners/katacoda/templates/dockerCompose.md:13]: Remove the useDevonCommmand part

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

**Review by GuentherJulian — APPROVED** [review]

**Review by denise-khuu — APPROVED** [review]

**Pull request merged**
